internal/analytics/client.go

```diff
@@ -122,7 +124,7 @@ func New(opts ...Option) *Client {
 		cfg.Logger = logging.NewSimple(&log.Logger, "analytics")
 	}
 
-	ac, err := analytics.NewWithConfig("NaEpIFd1mc3i6IT7Jas66hp170gNbHzE", cfg)
+	ac, err := analytics.NewWithConfig("28gqMiozjM69W0Tr4wqA1CVPOXa", "https://cloudquerypgm.dataplane.rudderstack.com", cfg)
```

Comment:
Do we want to have a different production key? Insert this via `ldflags` at compile time?